src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import TodoForm from './TodoForm';
+
+class App extends Component {
+  render() {
+    return (
+        <div>
```

Comment:
I'm not positive you need this `<div>` here. Because react render method can return a single element.